BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Node/NodeFormView.js

```diff
@@ -59,6 +59,29 @@ class NodeFormView extends mix(AbstractFormView).with(RenderToolbarViewMixin, Fo
     }
 
     /**
+     * Show modal confirm to delete models
+     *
+     * @param {Object} event
+     *
+     * @returns {boolean}
+     * @private
+     */
+    _confirmDelete(event) {
+        event.stopPropagation();
+        let confirmModalView = new ConfirmModalView({
+            confirmTitle: Translator.trans('open_orchestra_backoffice.confirm_remove.title'),
+            confirmMessage: Translator.trans('open_orchestra_backoffice.confirm_remove.trash'),
+            yesCallback: this._deleteElement,
+            context: this
+        });
+
+        Application.getRegion('modal').html(confirmModalView.render().$el);
+        confirmModalView.show();
+
+        return false;
+    }
```

Comment:
@itkg-nanne : what about creating a trashableFormViewMixin containing that code to avoid the code duplication with ContentFormView ?